rollup.config.js

```diff
@@ -64,7 +70,7 @@ export default {
     }),
     // copy assets into bundle
     copy({
```

Comment:
Can we remove this? I was the one adding it but I believe it's not needed after all